stylesheets/_palette.base.scss

```diff
@@ -9,6 +9,11 @@
 \* -------------------------------------------------------------------------- */
 
 $palette-branding: (
+    jadu-purple: (
+        darkest:    rgb(46, 54, 112),
+        dark:       rgb(51, 60, 121),
+        base:       rgb(61, 71, 150)
+    ),
```

Comment:
do we need a light and pale versions as well?

Reply:
Apologies, I'd done them after Lee sent me the branding guidelines but not committed them 🤦‍♂